### DISCUSSION TITLE
How to disable case insensitivity while using interop?

### DISCUSSION THREAD

**Discussion by scarletquasar** [sebastienros/jint#1783] [Q&A] [answered]
I am passing an instance to SetValue but both method options (ex: log and Log) seems to be available. I only want the camelCase option. Which configuration should I use to achieve that? Ps: Changing the method name in C# to camelCase have no effect.

**Comment by lahma** [comment, marked as answer]
Example of strict matching, you can write your own comparer to do strict first-lower.

https://github.com/sebastienros/jint/blob/4fe84ea436804a9f7eeedb00cc62d706d61f22c6/Jint.Tests/Runtime/InteropTests.cs#L2674-L2707
